client/ayon_maya/api/lib_rendersettings.py

```diff
@@ -48,7 +48,8 @@ def __init__(self, project_settings=None):
             "vray": render_settings["vray_renderer"]["image_prefix"],
             "arnold": render_settings["arnold_renderer"]["image_prefix"],
             "renderman": render_settings["renderman_renderer"]["image_prefix"],
-            "redshift": render_settings["redshift_renderer"]["image_prefix"]
+            "redshift": render_settings["redshift_renderer"]["image_prefix"],
+            "mayahardware2": render_settings["mayahardware_renderer"]["image_prefix"],
```

Comment:
I'm just wondering whether we should turn the settings key into `mayahardware2_renderer` just for consistency sake. Thoughts?

Reply:
we can somehow hardcode or leave it empty

Reply:
Sorry - what do you mean with leave empty?
```suggestion
            "mayahardware2": render_settings["mayahardware2_renderer"]["image_prefix"],
```
I mostly meant this :)

Reply:
This being in settings does make absolute settings - so the implementation is good. It's just a cosmetic note.